File: app/src/main/java/yiyo/gitlabandroid/mvp/views/LoginView.java
package yiyo.gitlabandroid.mvp.views;

import retrofit.RetrofitError;

/**
 * Created by yiyo on 11/07/15.
 */
public interface LoginView extends BaseView {

    void showProgress();

    void hideProgress();

    void setupUsernameError(String usernameError);

    void setupPasswordError(String passwordError);

    void navigateToHome();

    void showConnectionError(RetrofitError error);
}

File: app/src/main/kotlin/yiyo/gitlabandroid/mvp/views/LoginView.kt
package yiyo.gitlabandroid.mvp.views

/**
 * Created by yiyo on 11/07/15.
 */
interface LoginView : BaseView {

    fun showProgress()

    fun hideProgress()

    fun setupUsernameError(usernameError: String?)

    fun setupPasswordError(passwordError: String?)

    fun navigateToHome()

//    fun showConnectionError(error: RetrofitError)
}
